### PR TITLE
Feat buttons

### PR DESCRIPTION
**Pull request by Samarth-5109**
added primary button, button with an outline, link button, secondary button, icon button, button with icon.
Screenshot (35)

added float button
Screenshot (37)

### CODE REVIEW

File: components/buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,178 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+
+
+:root{
+    --primary-color:#456568;
+    --secondary-color:whitesmoke;
```
Comment (ZuberDunge) [inline components/buttons/button.css:6]: suggestion : you can shift this to base.css

File: components/buttons/button.css [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
```
Comment (ZuberDunge) [inline components/buttons/button.css:13]: suggestion : you can shift this to reset.css

File: styles/landing.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+
+
+:root{
+    --primary-color:#456568;
+    --background-color:#F9F9F9;
+
```
Comment (ZuberDunge) [inline styles/landing.css:7]: you can shift this to base.css, so that you don't have to define it in every css file

File: styles/landing.css [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    --primary-color:#456568;
+    --background-color:#F9F9F9;
+    
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
```
Comment (ZuberDunge) [inline styles/landing.css:14]: same here

File: styles/landing.css [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+.btn:hover{
+    background-color: #fff;
+    color:#456268;
+}
+
+.footer-container{
+    text-align: center;
+    margin-top: 5rem;
+    background-color: #456268;
```
Comment (ZuberDunge) [inline styles/landing.css:114]: use variables for colors

**Review by ZuberDunge — APPROVED** [review]
added few suggestion check them out, other than that, everything looks fine

File: components/buttons/button.css [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+    display: flex;
+    align-items: center;
+    background-color: var(--primary-color);
+    font-family: 'Roboto', sans-serif;
+    font-weight: bold;
+}
+.branding{
+    display: flex;
+    align-items: center;
+    /* gap:1rem; */
```
Comment (parulgupta2001) [inline components/buttons/button.css:33]: remove extra code

**Review by parulgupta2001 — APPROVED** [review]
remove extra commented code

**Pull request merged**